### ISSUE TITLE
[Bug]: App completely unusable: Update to 0.0.22

### ISSUE THREAD

**Issue by lcoogan** [snapcrafters/discord#140]
### What happened?

The app is completely unusable as Discord refuses to start unless it's on the latest version. Please update the snap.

### What should have happened?

The app should open as normal.

### Output of `snap info discord`

```shell
Unnecessary.
```

### Output of `snap connections discord`

```shell
Unnecessary.
```

### Output of `snap version`

```shell
Unnecessary.
```

**Comment by kenvandine** [comment]
I can't reproduce this with the latest in the stable channel (version 0.0.21 rev145)

It starts and works just fine.

**Comment by lcoogan** [comment]
Huh - earlier on I ran the app and got this window (not this exact image, I googled to find this one):
![image](https://user-images.githubusercontent.com/34213592/207986475-2e1a7e56-9fa8-4dfc-ab79-46d3013308d0.png)
I restarted the app and got the same message. By comparing the version that it was asking me to install (0.0.22) and the one in the snap store stable channel (0.0.21), I figured the issue was an update was needed on your side.

After seeing your message, I just started the app there and Discord said 'Downloading Updates' and then launched perfectly fine. Seemingly nothing has changed since I last tried (I just checked `snap info discord` and it hasn't updated or anything). I guess I'll close the bug then? Thanks for your help anyway.
